ang/civicase/case/search/directives/search.directive.html

```diff
@@ -20,24 +20,29 @@ <h3 class="civicase__case-filter-panel__title"> {{ pageTitle }} </h3>
         <label >{{ ts('Case ID') }}</label>
         <input class="form-control" type="number" ng-model="filters.id" />
       </div>
-      <div ng-if="isEnabled('contact_id')" class="civicase__case-filter-form-elements">
-        <label>{{ ts('Contact Search') }}</label>
-        <input class="form-control"
-          ng-model="searchForm.selectedContacts"
-          crm-entityref="{
-            entity: 'Contact',
-            select: { multiple: true }
-          }"
-        />
-        <input class="form-control"
-          ng-model="searchForm.selectedContactRoles"
-          crm-ui-select="{
-            allowClear: true,
-            multiple: true,
-            placeholder: 'Contact role',
-            data: contactRoleFilters
-          }"
-        />
+      <div ng-if="isEnabled('contact_id')" class="civicase__case-filter-form-elements row">
```

Comment:
We have changed the class name, but I dont see any css being changed. Is this working fine?